test/Product_DelimitedTable_INVALID_lid_ref_no-match.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.sch"
+  schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?xml-model href="../PDS4_PROC_1L00_1400.sch"
+    schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<Product_Observational xmlns="http://pds.nasa.gov/pds4/pds/v1"
+    xmlns:pds="http://pds.nasa.gov/pds4/pds/v1"
+    xmlns:proc="http://pds.nasa.gov/pds4/proc/v1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.xsd
+    http://pds.nasa.gov/pds4/proc/v1 ../PDS4_PROC_1L00_1400.xsd">  
+    
+    <Identification_Area>
+        <logical_identifier>urn:nasa:pds:example.dph.sampleproducts:exampleproducts:streamdelimited.co-s-inst-2-dummy-data-v1.0</logical_identifier>
+        <version_id>1.0</version_id>
+        <title>CASSINI Orbiter Instrument Experiment</title>
+        <information_model_version>1.21.0.0</information_model_version>

```

Comment:
Per https://github.com/pds-data-dictionaries/ldd-proc/pull/28#discussion_r1376617766, there are some rules that need to be applied to these test data, which is why, unfortunately, these LDDs are not generating properly.